### PR TITLE
Change few variable names

### PR DESCRIPTION
**Pull request by AdilKhurshid**
Implemented put get post and delete and change few variable name

### CODE REVIEW

File: CustomServer/swagger/swagger.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ paths:
           required: false
```
Comment (kashifali94) [inline CustomServer/swagger/swagger.yml]: what does required false means here ?

File: CustomServer/swagger/swagger.yml [hunk 2]
```diff
@@ -79,7 +79,7 @@ paths:
...
```
Comment (kashifali94) [inline CustomServer/swagger/swagger.yml]: same

**Review by kashifali94 — CHANGES_REQUESTED** [review]
in post and patch you have used required false what is the purpose ? and if they are false so what are the values which can be sent in request body ?

File: CustomServer/swagger/swagger.yml (hunk 1 above)
Comment (AdilKhurshid) [reply CustomServer/swagger/swagger.yml]: its mean these fields are optional. we can hit api even without providing these query parameters

**Comment by AdilKhurshid** [comment]
they will send no value. In API I can assign any value to it, by default it assign zero. By using required = false it send value not pointer

**Pull request merged**